packages/react-instantsearch/src/core/createConnector.js

```diff
@@ -4,6 +4,19 @@ import {omit, has} from 'lodash';
 import {shallowEqual, getDisplayName} from './utils';
 
 /**
+ * @typedef {object} ConnectorDescription
```

Comment:
Unsure, if it currently does not have any output in our documentation website I would say we do not need it for now.